### PR TITLE
Absent by Design 翻译更新

### PR DESCRIPTION
**Pull request by ChuijkYahus**
极有可能审着审着就血压升高了

### CODE REVIEW

File: projects/1.20/assets/absent-by-design/absentbydesign/lang/zh_cn-composition.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                    "stone_bricks": "石砖",
+                    "stone": "石头"
+                }
+            ]
+        },
+        {
+            "templates": {
+                "block.absentbydesign.fence_log_{0}": "{0}原木栅栏",
+                "block.absentbydesign.wall_{0}_log": "{0}原木墙",
+                "block.absentbydesign.wall_{0}_planks": "{0}木墙",
```
Comment (SlimeSB) [inline projects/1.20/assets/absent-by-design/absentbydesign/lang/zh_cn-composition.json:143]: 木板墙。
规范之前已更新 #4525

Comment (ChuijkYahus) [reply projects/1.20/assets/absent-by-design/absentbydesign/lang/zh_cn-composition.json:143]: 这些的原文也没有`Planks`这个词啊

Comment (SlimeSB) [reply projects/1.20/assets/absent-by-design/absentbydesign/lang/zh_cn-composition.json:143]: "block.absentbydesign.wall_cherry_planks":"Cherry Planks Wall",
	"block.absentbydesign.wall_mangrove_planks": "Mangrove Planks Wall",
真头疼，别改了。

**Review by SlimeSB — APPROVED** [review]

File: projects/1.20/assets/absent-by-design/absentbydesign/lang/zh_cn-composition.json (hunk 1 above)
Comment (ChuijkYahus) [reply projects/1.20/assets/absent-by-design/absentbydesign/lang/zh_cn-composition.json:143]: > ```
> "block.absentbydesign.wall_cherry_planks":"Cherry Planks Wall",
> "block.absentbydesign.wall_mangrove_planks": "Mangrove Planks Wall",
> ```
> 
> 真头疼，别改了。

这模组的键名键值不规范算是祖传的了，就这样吧先

**Pull request merged**